src/index.css

```diff
@@ -1,13 +1,53 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.MainBody {
+  width: 400px;
+  background: rgb(192, 188, 188);
+  display: grid;
+  grid-template-rows: auto auto auto;
+  gap: 1px;
+  margin: 15% 2% 0 auto;
+}
+
+.result {
+  background: rgb(158, 156, 156);
+  display: flex;
+  flex-direction: row-reverse;
+  color: white;
+  padding: 4%;
+}
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓

Reply:
- [OPTIONAL] Kindly consider making this, I'm still seeing many style properties repetitions that could be refactor as requested by the previous reviiouwer 💯 👍🏻